contrib/cirrus/lib.sh

```diff
@@ -192,8 +200,38 @@ EOF
     cat $FILEPATH
```

Comment:
[not shown in github diff]. Please remove `install_ooe()` above.

Reply:
got it, good catch, thanks.